bin/codeclimate-ruby

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env ruby
+
+require "codeclimate-test-reporter"
+
+COVERAGE_FILE = "coverage/.resultset.json".freeze
+
+if ENV["CODECLIMATE_REPO_TOKEN"]
+  if File.exist?(COVERAGE_FILE)
+    result = JSON.parse(File.read(COVERAGE_FILE))
+
+    CodeClimate::TestReporter::Formatter.new.format(result)
+  else
+    $stderr.puts "Coverage results not found"
+    exit(1)
+  end
+else
+  $stderr.puts "Cannot post results: environment variable CODECLIMATE_REPO_TOKEN must be set."
+  exit(1)
```

Comment:
I might consider this a warn-but-still-exit-0 case, WDYT?

The other case, where you have the env var set (so you intend to submit) but we don't find a file, make sense to me as a non-zero exit.